src/main/kotlin/de/smartsquare/squit/util/ConfigExtensions.kt

```diff
@@ -142,6 +143,11 @@ val Config.headers
         .map { (key, value) -> key to value.unwrapped().toString() }
         .toMap()
 
+/**
+ * The http status response code that is expected in the response.
+ */
+val Config.responseCode get() = getSafeString(RESPONSE_CODE)
```

Comment:
Can we make this an `Int` and validate that it lies in the HTTP status code range (to prevent user errors as early as possible)?

Reply:
Yes